great_expectations/core/batch.py

```diff
@@ -304,10 +308,10 @@ def __init__(
         )
 
     @staticmethod
-    def _validate_batch_request(
+    def _validate_init_parameters(
         datasource_name: str,
         data_connector_name: str,
-        data_asset_name: str,
+        data_asset_name: str = None,
```

Comment:
Same concern as in the comment above. I have held the understanding that BatchRequest had to be strictly validated. Happy to discuss.

Reply:
I'll close this since it's been adjusted